### PR TITLE
List team members

### LINKED ISSUE #360: List Team Members
It is not a simple task to see if the user is a member of a team. There should be a command that lists the members and owners of the team (probably do not need to protect this one as outside users may want this information too).

### PR DESCRIPTION
**Pull request by KieranHons**
Adds new command which shows team owners and members for selected team.

Resolves #360 
screen shot 2018-09-11 at 11 38 52

### CODE REVIEW

**Comment by kbristow** [comment]
Is there a reason why members have are done as attachments whilst owners are just put in the text?

**Comment by KieranHons** [comment]
Thought it might help differentiate between the two more. Kind of shows a hierarchy

File: src/gluon/commands/team/ListTeamMembers.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    })
+    public teamName: string;
+
+    constructor(public gluonService = new GluonService(),
+                public axiosInstance = new AwaitAxios()) {
+        super();
+    }
+
+    protected async runCommand(ctx: HandlerContext) {
+        const result = await this.axiosInstance.get(`${QMConfig.subatomic.gluon.baseUrl}/teams/?name=${this.teamName}`);
```
Comment (kbristow) [inline src/gluon/commands/team/ListTeamMembers.ts]: Can you change this to using the gluon service? It should have functions to get team by team name. Then you can remove the axiosInstance from the class.

Comment (kbristow) [reply src/gluon/commands/team/ListTeamMembers.ts]: `const teamDetails = this.gluonService.gluonTeamByName(this.teamName);`. That should work.

**Pull request merged**